src/Semantic/Git.hs

```diff
@@ -17,62 +17,74 @@ module Semantic.Git
 
 import Prologue
 
-import           Data.Attoparsec.Text (Parser)
-import           Data.Attoparsec.Text as AP
+import           Data.Attoparsec.ByteString (Parser)
+import           Data.Attoparsec.ByteString as AP
+import           Data.ByteString (ByteString)
+import           Data.ByteString.Internal (w2c)
+import qualified Data.ByteString.UTF8 as UTF8
+import qualified Data.ByteString as B
 import qualified Data.ByteString.Streaming as ByteStream
+import qualified Data.Attoparsec.ByteString.Streaming as Streaming.Parse
 import           Data.Char
 import           Data.Either (fromRight)
 import           Data.Text as Text
-import           Shelly hiding (FilePath)
+import           Text.Parser.Combinators (sepEndBy)
 import qualified Streaming.Process
-import qualified System.Process as Process (proc)
+import qualified System.Process as Process
 import qualified Source.Source as Source
 
 -- | git clone --bare
 clone :: Text -> FilePath -> IO ()
-clone url path = sh $ do
-  run_ "git" ["clone", "--bare", url, pack path]
+clone url path = Process.callProcess "git"
+  ["clone", "--bare", Text.unpack url, path]
```

Comment:
We don’t need any streaming stuff here, since we just want to run it and wait for completion.